schema/spacewalk/common/tables/tables.deps

```diff
@@ -299,3 +299,4 @@ susePaygDimensionResult            :: billing_dimension_t susePaygDimensionCompu
 suseAppstream                      :: rhnChannel
 suseAppstreamPackage               :: suseAppstream rhnPackage
 suseAppstreamApi                   :: suseAppstream
+rhnChannelNewestPackage            :: suseAppstream
```

Comment:
I wonder why this does not exist already in the file?
It also require rhnChannel, rhnPackageName, rhnPackageEVR and rhnPackageArch.
Feel free to add them also to be complete.